src/components/NewGalaxyModal.js

```diff
@@ -61,17 +79,38 @@ export const NewGalaxyModal = _.flow(
       ['launchWarn', () => {
         return h(GalaxyLaunchButton, { app, onClick: onDismiss })
       }],
+      ['paused', () => {
+        return h(Fragment, [
+          h(ButtonPrimary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
+          h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: () => { resumeGalaxy() } }, ['Resume'])
+        ])
+      }],
       [Utils.DEFAULT, () => {
         return !!app ?
           h(Fragment, [
             h(ButtonSecondary, { style: { marginRight: 'auto' }, onClick: () => setViewMode('deleteWarn') }, ['Delete']),
-            h(ButtonPrimary, { onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
+            (app.status === 'RUNNING') && h(ButtonSecondary, { style: { marginRight: '1rem' }, onClick: () => { pauseGalaxy() } }, ['Pause']),
+            (app.status === 'STOPPED') && h(ButtonPrimary, { style: { marginRight: '1rem' }, onClick: () => { resumeGalaxy() } }, ['Resume']),
+            (app.status === 'RUNNING') && h(ButtonPrimary, { onClick: () => setViewMode('launchWarn') }, ['Launch Galaxy'])
           ]) :
           h(ButtonPrimary, { onClick: () => setViewMode('createWarn') }, ['Next'])
       }]
     )
   }
 
+  const renderMessaging = () => {
+    return Utils.cond(
+      [viewMode === 'createWarn', renderCreateWarning],
+      [viewMode === 'deleteWarn', renderDeleteWarning],
+      [viewMode === 'launchWarn', renderLaunchWarning],
+      [!!app && (app.status === 'STOPPED'), renderPaused],
+      [!!app && (app.status === 'STOPPING'), renderStoppingWarning],
```

Comment:
Probably this should also be for the "pre-stopping" case since that's basically a part of stopping

Reply:
```suggestion
      [!!app && (app.status === 'STOPPING' || app.status === ''), renderStoppingWarning],
```

> It looks like leo doesn't allow deleting while Galaxy is stopping? I am getting this error when trying
> <img alt="Screen Shot 2021-01-08 at 9 59 11 AM" width="1213" src="https://user-images.githubusercontent.com/54322292/104029788-6a959980-5198-11eb-827a-8608fd4bb9db.png">

Are you planning to make the change for allowing deletion in stopped/stopping on leo side with this ticket?